hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ScheduledExecutorCodecTemplate.java

```diff
@@ -42,117 +42,203 @@
      * Submits the task to partition for execution, partition is chosen based on multiple criteria of the given task.
      *
      * @param schedulerName The name of the scheduler.
-     * @param taskDefinition  The data form of the Callable or Runnable task to be executed on that scheduler
+     * @param type type of schedule logic, values 0 for SINGLE_RUN, 1 for AT_FIXED_RATE
+     * @param name The name of the task
+     * @param task The data form of the Callable or Runnable task to be executed on that scheduler
+     * @param initialDelayInMillis initial delay in milliseconds
+     * @param periodInMillis period between each run in milliseconds
      */
     @Since("1.4")
     @Request(id = 2, retryable = true, response = ResponseMessageConst.VOID, partitionIdentifier = "partitionId")
-    void submitToPartition(String schedulerName, Data taskDefinition);
+    void submitToPartition(String schedulerName, byte type, String name, Data task, long initialDelayInMillis, long periodInMillis);
 
     /**
      * Submits the task to a member for execution, member is provided in the form of an address.
      *
      * @param schedulerName The name of the scheduler.
-     * @param taskDefinition  The data form of the Callable or Runnable task to be executed on that scheduler
      * @param address The address of the member where the task will get scheduled.
+     * @param type type of schedule logic, values 0 for SINGLE_RUN, 1 for AT_FIXED_RATE
+     * @param name The name of the task
```

Comment:
No, I did not. It is name of the task. 4. parameter.
schedularName is a different parameter. 1. parameter